pandas/io/common.py

```diff
@@ -267,6 +267,16 @@ def urlopen(*args, **kwargs):
     """
     import urllib.request
 
+    if "storage_options" in kwargs and kwargs["storage_options"] is not None:
```

Comment:
This shouldn't be needed. This is already handled in `_get_filepath_or_buffer`

Reply:
Right, I missed that. I've reverted the implementation of `urlopen()` to its original form, and I've used `common.get_handle()` in `html.py` instead.